scene/2d/collision_object_2d.cpp

```diff
@@ -589,6 +589,10 @@ PackedStringArray CollisionObject2D::get_configuration_warnings() const {
 		warnings.push_back(RTR("This node has no shape, so it can't collide or interact with other objects.\nConsider adding a CollisionShape2D or CollisionPolygon2D as a child to define its shape."));
 	}
 
+	if (!(get_transform().is_conformal() && get_global_transform().is_conformal())) {
```

Comment:
```suggestion
	if (!get_transform().is_conformal() || !get_global_transform().is_conformal()) {
```
I'd suggest for readability to not make a grouped negated check but instead a direct check everywhere here